quilt/tools/main.py

```diff
@@ -96,7 +97,8 @@ def check_hash(group, hashstr):
     tag_remove_p.set_defaults(func=command.tag_remove)
 
     install_p = subparsers.add_parser("install")
-    install_p.add_argument("package", type=str, help="owner/package_name[/path/...]")
+    install_p.add_argument("package", type=str, help="owner/package_name[/path/...] or @filename",
+                           nargs="?", default="@quilt.yml")
```

Comment:
Now that you've made 'quilt.yml' a constant in const.py, you should use that constant here.